src/index.ts

```diff
@@ -111,16 +120,20 @@ const _init = (inputOptions: SnippetOptions, readyCallback?: ReadyCallback) => {
     if (!options.script) {
       options.script = 'edge.fullstory.com/s/fs-debug.js';
     } else {
-      console.warn('Ignoring `debug = true` because `script` is set');
+      log('Ignoring `debug = true` because `script` is set', options.disableLogs);
     }
   }
 
+  if (options.disableLogs) {
+    window._fs_disable_logs = true;
```

Comment:
Then this could be something like:
```
  if (options.disableLogs) {
    _fsDisableSdkLogs = options.disableLogs;
  }
```

Reply:
Hey, thanks for the feedback. I have made the necessary changes. Can you please have a look? BTW,  I see npm audit issues in test-deploy workflow.